composer.json

```diff
@@ -12,12 +12,11 @@
     "require": {
         "php": "^7.2|^8.0",
         "ramsey/uuid": "^3|^4",
-        "symfony/yaml": "^3.2|^4.0|^5.0"
```

Comment:
Like I said above, the `prefer-lowest` build showed that this version was incompatible with PHP 8. I thus removed it here given the age from symfony/yaml. We could look into re-adding it and juggling around the matrix build with the yaml dependency but I'll leave that choice up to you. Let me know what you want to do.